packages/policy/src/changer.ts

```diff
@@ -55,108 +51,95 @@ async function cachedGet(url: string) {
   return contents;
 }
 
-/**
- * Submit a pull request to the target repository to add a CODE_OF_CONDUCT.md.
- * @param owner Repostiory owner - ex: "googleapis"
- * @param repo Repository name - ex: "repo-automation-bots"
- * @param octokit Pre-authenticated octokit instance
- */
-export async function addCodeOfConduct(repo: GitHubRepo, octokit: Octokit) {
-  // first, make sure there's no open PR for this
-  const title = 'chore: add a Code of Conduct';
-  const prs = await octokit.search.issuesAndPullRequests({
-    q: `repo:${repo.full_name} "${title}" in:title is:open`,
-  });
-  if (prs.data.total_count > 0) {
-    return;
+export class Changer {
```

Comment:
nit: I feel like `Changer` is a bit generic, perhaps `MissingFilesSubmitter`, or something along those lines (_a a great name didn't immediately jump out at me_).